magpie/__init__.py

```diff
@@ -36,18 +36,19 @@ def __init__(self, keras_model=None, word2vec_model=None, scaler=None,
             self.scaler = scaler
 
     def train(self, train_dir, vocabulary, test_dir=None, callbacks=None,
-              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE,
+              nn_model=NN_ARCHITECTURE, batch_size=BATCH_SIZE, test_split=0.0,
```

Comment:
It's the fraction of the data to use as held-out test data, so if it's 0.1 then then 10% of all samples are used for testing. The default is 0.0 which means all of the data is used only for training. Do you think `test_ratio` would be a better name?